### ISSUE TITLE
Failure of Success Criterion 1.3.1 due to using th elements

### ISSUE THREAD

**Issue by veselints** [w3c/wcag#2620]
Hi there!

I have a question. If we have the following HTML structure:
```html
<div role="grid">
  <table role="presentation">
    <tr role="row">
      <th role="columnheader">Header</th>
    </tr>
    <tr role="row">
      <td role="gridcell">data</td>
    </tr>
  </table>
</div>
```

does the 1.3.1 ([using th elements](https://www.w3.org/WAI/WCAG21/Techniques/failures/F46.html)) applies?

My understanding is that it does not because the above is a data table (and not a layout table) even if `role=presentation` is used on the table element. The accessibility tree builds the following hierarchy which represents a data table:
```
grid
    row
        columheader
    row
        gridcell
```

**Comment by JAWS-test** [comment]
Your source code does not represent a violation of SC 1.3.1 in my opinion. F46 does not apply here if the content is a grid and can be operated accordingly (https://www.w3.org/WAI/ARIA/apg/patterns/grid/).

It would be easier to use role=grid not on the div, but on the table. Then the other grid roles within the table would not be necessary.

**Comment by alastc** [comment]
I agree with @JAWS-test, WCAG is interested in the programmatic representation delivered, not how that is achieved. (Obviously some techniques can be more robust than others, but if it works, that's ok.)

**Comment by veselints** [comment]
Thank you very much for your input! After clarifying the case, I believe I should close the issue.
